packages/edit-site/src/components/layout/style.scss

```diff
@@ -4,6 +4,16 @@
 	color: $white;
 	display: flex;
 	flex-direction: column;
+
+	// expand the fixed block toolbar to cover the document title control
```

Comment:
Personal pet peeve, I have a preference for sentence case and a period at the end.